IdentityCore/src/oauth2/aad_base/MSIDAADOauth2Factory.m

```diff
@@ -276,6 +276,7 @@ - (BOOL)fillBaseToken:(MSIDBaseToken *)baseToken
     return YES;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->